---
id: 3PbjxNe7yIn2I9yKJfqXD
title: Papers
desc: ''
updated: 1647777821106
created: 1610194509817
---


# Readlist of publications


This is a dump.
Entries here should be sent to their respective dendron notes when treated.


# To Embed or Not: Network Embedding as a Paradigm in Computational Biology

REVIEW article
Front. Genet., 01 May 2019 | https://doi.org/10.3389/fgene.2019.00381


https://www.frontiersin.org/articles/10.3389/fgene.2019.00381/full

# Crossroads in the evolution of plant specialized metabolism

https://www.sciencedirect.com/science/article/pii/S1084952122000738


# A knowledge graph to interpret clinical proteomics data


https://www.nature.com/articles/s41587-021-01145-6

# Infrastructure and Population of the OpenBiodiv Biodiversity Knowledge Graph

https://bdj.pensoft.net/article/67671/


# Reshaping the Academic Self: Connecting Education & Open Science

Sunday 05 September 2021

https://zenodo.org/record/5345573#.YTSAwDZfiDW



Monday 14 June 2021

# Functional biology in its natural context: A search for emergent simplicity

https://elifesciences.org/articles/67646

Monday 31 May 2021

# A deep generative model enables automated structure elucidation of novel psychoactive substances

https://chemrxiv.org/articles/preprint/A_Deep_Generative_Model_Enables_Automated_Structure_Elucidation_of_Novel_Psychoactive_Substances/14644854/1


Thursday 27 May 2021

# Multiomic Big Data Analysis Challenges: Increasing Confidence in the Interpretation of Artificial Intelligence Assessments
https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c04850


Tuesday 25 May 2021


# Identifying molecules as biosignatures with assembly theory and mass spectrometry

https://www.nature.com/articles/s41467-021-23258-x
> 
> The search for alien life is hard because we do not know what signatures are unique to life. We show why complex molecules found in high abundance are universal biosignatures and demonstrate the first intrinsic experimentally tractable measure of molecular complexity, called the molecular assembly index (MA). To do this we calculate the complexity of several million molecules and validate that their complexity can be experimentally determined by mass spectrometry. This approach allows us to identify molecular biosignatures from a set of diverse samples from around the world, outer space, and the laboratory, demonstrating it is possible to build a life detection experiment based on MA that could be deployed to extraterrestrial locations, and used as a complexity scale to quantify constraints needed to direct prebiotically plausible processes in the laboratory. Such an approach is vital for finding life elsewhere in the universe or creating de-novo life in the lab.

# Supervised topic modeling for predicting molecular substructure from mass spectrometry [version 1; peer review: awaiting peer review]

https://f1000research.com/articles/10-403

> Small-molecule metabolites are principal actors in myriad phenomena
> across biochemistry and serve as an important source of biomarkers
> and drug candidates. Given a sample of unknown composition,
> identifying the metabolites present is difficult given the large number
> of small molecules both known and yet to be discovered. Even for
> biofluids such as human blood, building reliable ways of identifying
> biomarkers is challenging. A workhorse method for characterizing
> individual molecules in such untargeted metabolomics studies is
> tandem mass spectrometry (MS/MS). MS/MS spectra provide rich
> information about chemical composition. However, structural
> characterization from spectra corresponding to unknown molecules
> remains a bottleneck in metabolomics. Current methods often rely on
> matching to pre-existing databases in one form or another.  Here we
> develop a preprocessing scheme and supervised topic modeling
> approach to identify modular groups of spectrum fragments and
> neutral losses corresponding to chemical substructures using labeled
> latent Dirichlet allocation (LLDA) to map spectrum features to known
> chemical structures. These structures appear in new unknown spectra
> and can be predicted. We find that LLDA is an interpretable and
> reliable method for structure prediction from MS/MS spectra.
> Specifically, the LLDA approach has the following advantages: (a)
> molecular topics are interpretable; (b) A practitioner can select any set
> of chemical structure labels relevant to their problem; (c ) LLDA
> performs well and can exceed the performance of other methods in
> predicting substructures in novel contexts.


Tuesday 11 May 2021

# Plant scientists’ research attention is skewed towards colourful, conspicuous and broadly distributed flowers

https://www.nature.com/articles/s41477-021-00912-2


Tuesday 20 April 2021

# Repository scale classification and decomposition of tandem mass spectral data
Scientific Reports volume 11, Article number: 8314 (2021) Cite this article

https://www.nature.com/articles/s41598-021-87796-6


Monday 29 March 2021

# Capturing scientific knowledge in computable form

https://biofactoid.org

https://www.biorxiv.org/content/10.1101/2021.03.10.382333v1



# First known gene transfer from plant to insect identified

https://www.nature.com/articles/d41586-021-00782-w



Friday 05 March 2021


# Connecting molecular sequences to their voucher specimens

https://biohackrxiv.org/93qf4/

[[#wikidata|tag.wikidata]]


2021-02-10 10:13

# Large-scale tandem mass spectrum clustering using fast nearest neighbor searching


https://www.biorxiv.org/content/10.1101/2021.02.05.429957v1

[[#spectral_comparison|tag.spectral_comparison]]

2021-02-09 11:45

# NetLSD: Hearing the Shape of a Graph

Graph comparison an analysis at large scale

https://arxiv.org/pdf/1805.10712.pdf

[[#graph|tag.graph]]


2021-02-08 19:42

# Automated assembly of a reference taxonomy for phylogenetic data synthesis

The original OTOL paper
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515096/

[[#taxo|tag.taxo]] 


2021-02-03 16:51

# Data integration enables global biodiversity synthesis

https://www.pnas.org/content/118/6/e2018093118

[[#biodiversity|tag.biodiversity]] [[#gbif|tag.gbif]]

2021-01-31 20:19


# Interoperable chemical structure search service

https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0367-2




2021-01-29 19:22

# Mapping the space of chemical reactions using attention-based neural networks

Reymond's paper on reactions based tmaps. Try this on BNICE set of react or on MetaNetX

https://www.nature.com/articles/s42256-020-00284-w.epdf?sharing_token=CGHu6tcqBiFmC1nRfnOrxNRgN0jAjWel9jnR3ZoTv0NiJSBmQfyOiLfbwen1TgEszNU5Xao_0Gs7D0u8tLUOJxaLAUc469WZKWG5K9wsqIe98dZOREVhQ33gqcr33AgNUjnp4cVXojp4aKe2xCbWtNaxOZtpOjBOsdX3O8yxVkI%3D




2021-01-24 17:04

# Estimate Metabolite Taxonomy and Structure with a FragmentCentered Database and Fragment Network


https://arxiv.org/pdf/2101.03784.pdf




2021-01-17 09:21

# Network-based strategies in metabolomics data analysis and interpretation: from molecular networking to biological interpretation

https://www.tandfonline.com/doi/full/10.1080/14789450.2020.1766975

[[networks]]




2021-01-13 22:12

# Towards a biodiversity knowledge graph

https://riojournal.com/articles.php?journal_name=rio&id=8767
Check RIO in general
Dig into http://beta.briefideas.org/
//TODO Make a nanopub and related  note


2021-01-12 16:41

# Metabolite networks filtering and explo

https://www.biorxiv.org/content/biorxiv/early/2021/01/06/2021.01.06.425569.full.pdf


2021-01-09 14:18

# Florian Huber blog post msmsmatch tuto
https://blog.esciencecenter.nl/build-your-own-mass-spectrometry-analysis-pipeline-in-python-using-matchms-part-i-d96c718c68ee



2021-01-09 13:15

# Chemical graph generators
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008504




This resource represents an important advancement in the design and deployment of a comprehensive and collaborative natural products knowledge base.
