<!DOCTYPE html><html><head><meta name="viewport" content="width=device-width"/><meta charSet="utf-8"/><link rel="icon" href="/favicon.ico"/><title>Papers</title><meta name="robots" content="index,follow"/><meta name="googlebot" content="index,follow"/><meta name="description" content="Personal knowledge space"/><meta property="og:title" content="Papers"/><meta property="og:description" content="Personal knowledge space"/><meta property="og:url" content="https://oolonek.github.io/dendron-ws-public/notes/3PbjxNe7yIn2I9yKJfqXD/"/><meta property="og:type" content="article"/><meta property="article:published_time" content="1/9/2021"/><meta property="article:modified_time" content="2/1/2022"/><link rel="canonical" href="https://oolonek.github.io/dendron-ws-public/notes/3PbjxNe7yIn2I9yKJfqXD/"/><meta name="next-head-count" content="14"/><link rel="preload" href="/dendron-ws-public/_next/static/css/30a05f83f6733b1f.css" as="style"/><link rel="stylesheet" href="/dendron-ws-public/_next/static/css/30a05f83f6733b1f.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/dendron-ws-public/_next/static/chunks/polyfills-5cd94c89d3acac5f.js"></script><script src="/dendron-ws-public/_next/static/chunks/webpack-1626b79bda42e25e.js" defer=""></script><script src="/dendron-ws-public/_next/static/chunks/framework-dc33c0b5493501f0.js" defer=""></script><script src="/dendron-ws-public/_next/static/chunks/main-991ccdf7687f671a.js" defer=""></script><script src="/dendron-ws-public/_next/static/chunks/pages/_app-90b9f496cfcf1b3c.js" defer=""></script><script src="/dendron-ws-public/_next/static/chunks/155-0a8d44b6408c244d.js" defer=""></script><script src="/dendron-ws-public/_next/static/chunks/pages/notes/%5Bid%5D-30a63e62504f2dc5.js" defer=""></script><script src="/dendron-ws-public/_next/static/Fif_h8o_yalnGTkyFx1wu/_buildManifest.js" defer=""></script><script src="/dendron-ws-public/_next/static/Fif_h8o_yalnGTkyFx1wu/_ssgManifest.js" defer=""></script><script src="/dendron-ws-public/_next/static/Fif_h8o_yalnGTkyFx1wu/_middlewareManifest.js" defer=""></script></head><body><div id="__next" data-reactroot=""><section class="ant-layout" style="width:100%;min-height:100%"><header class="ant-layout-header" style="position:fixed;isolation:isolate;z-index:1;width:100%;border-bottom:1px solid #d4dadf;height:64px;padding:0 24px 0 2px"><div class="ant-row ant-row-center" style="max-width:992px;justify-content:space-between;margin:0 auto"><div style="display:flex" class="ant-col ant-col-xs-20 ant-col-sm-4"></div><div class="ant-col gutter-row ant-col-xs-0 ant-col-sm-20 ant-col-md-20 ant-col-lg-19"><div class="ant-select ant-select-lg ant-select-auto-complete ant-select-single ant-select-allow-clear ant-select-show-search" style="width:100%"><div class="ant-select-selector"><span class="ant-select-selection-search"><input type="search" autoComplete="off" class="ant-select-selection-search-input" role="combobox" aria-haspopup="listbox" aria-owns="undefined_list" aria-autocomplete="list" aria-controls="undefined_list" aria-activedescendant="undefined_list_0" value=""/></span><span class="ant-select-selection-placeholder">For full text search please use the &#x27;?&#x27; prefix. e.g. ? Onboarding</span></div></div></div><div style="display:none;align-items:center;justify-content:center" class="ant-col ant-col-xs-4 ant-col-sm-4 ant-col-md-0 ant-col-lg-0"><span role="img" aria-label="menu" style="font-size:24px" tabindex="-1" class="anticon anticon-menu"><svg viewBox="64 64 896 896" focusable="false" data-icon="menu" width="1em" height="1em" fill="currentColor" aria-hidden="true"><path d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"></path></svg></span></div></div></header><section class="ant-layout site-layout" style="margin-top:64px"><section class="ant-layout site-layout" style="flex-direction:row"><section class="ant-layout site-layout-sidebar" style="flex:0 0 auto;width:calc(max((100% - 992px) / 2, 0px) + 200px);min-width:200px;padding-left:calc((100% - 992px) / 2)"><aside class="ant-layout-sider ant-layout-sider-dark" style="position:fixed;overflow:auto;height:calc(100vh - 64px);flex:0 0 200px;max-width:200px;min-width:200px;width:200px"><div class="ant-layout-sider-children"></div></aside></section><section class="ant-layout side-layout-main" style="max-width:1200px;display:initial"><main class="ant-layout-content main-content" role="main" style="padding:0 24px"><div class="ant-row"><div class="ant-col ant-col-24"><div class="ant-row" style="margin-left:-10px;margin-right:-10px"><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-24 ant-col-md-18"><div><h1 id="papers"><a aria-hidden="true" class="anchor-heading" href="#papers"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Papers</h1>
<h1 id="readlist-of-publications"><a aria-hidden="true" class="anchor-heading" href="#readlist-of-publications"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Readlist of publications</h1>
<p>This is a dump.
Entries here should be sent to their respective dendron notes when treated.</p>
<h1 id="a-knowledge-graph-to-interpret-clinical-proteomics-data"><a aria-hidden="true" class="anchor-heading" href="#a-knowledge-graph-to-interpret-clinical-proteomics-data"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>A knowledge graph to interpret clinical proteomics data</h1>
<p><a href="https://www.nature.com/articles/s41587-021-01145-6">https://www.nature.com/articles/s41587-021-01145-6</a></p>
<h1 id="infrastructure-and-population-of-the-openbiodiv-biodiversity-knowledge-graph"><a aria-hidden="true" class="anchor-heading" href="#infrastructure-and-population-of-the-openbiodiv-biodiversity-knowledge-graph"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Infrastructure and Population of the OpenBiodiv Biodiversity Knowledge Graph</h1>
<p><a href="https://bdj.pensoft.net/article/67671/">https://bdj.pensoft.net/article/67671/</a></p>
<h1 id="reshaping-the-academic-self-connecting-education--open-science"><a aria-hidden="true" class="anchor-heading" href="#reshaping-the-academic-self-connecting-education--open-science"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Reshaping the Academic Self: Connecting Education &#x26; Open Science</h1>
<p>Sunday 05 September 2021</p>
<p><a href="https://zenodo.org/record/5345573#.YTSAwDZfiDW">https://zenodo.org/record/5345573#.YTSAwDZfiDW</a></p>
<p>Monday 14 June 2021</p>
<h1 id="functional-biology-in-its-natural-context-a-search-for-emergent-simplicity"><a aria-hidden="true" class="anchor-heading" href="#functional-biology-in-its-natural-context-a-search-for-emergent-simplicity"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Functional biology in its natural context: A search for emergent simplicity</h1>
<p><a href="https://elifesciences.org/articles/67646">https://elifesciences.org/articles/67646</a></p>
<p>Monday 31 May 2021</p>
<h1 id="a-deep-generative-model-enables-automated-structure-elucidation-of-novel-psychoactive-substances"><a aria-hidden="true" class="anchor-heading" href="#a-deep-generative-model-enables-automated-structure-elucidation-of-novel-psychoactive-substances"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>A deep generative model enables automated structure elucidation of novel psychoactive substances</h1>
<p><a href="https://chemrxiv.org/articles/preprint/A_Deep_Generative_Model_Enables_Automated_Structure_Elucidation_of_Novel_Psychoactive_Substances/14644854/1">https://chemrxiv.org/articles/preprint/A_Deep_Generative_Model_Enables_Automated_Structure_Elucidation_of_Novel_Psychoactive_Substances/14644854/1</a></p>
<p>Thursday 27 May 2021</p>
<h1 id="multiomic-big-data-analysis-challenges-increasing-confidence-in-the-interpretation-of-artificial-intelligence-assessments"><a aria-hidden="true" class="anchor-heading" href="#multiomic-big-data-analysis-challenges-increasing-confidence-in-the-interpretation-of-artificial-intelligence-assessments"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Multiomic Big Data Analysis Challenges: Increasing Confidence in the Interpretation of Artificial Intelligence Assessments</h1>
<p><a href="https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c04850">https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c04850</a></p>
<p>Tuesday 25 May 2021</p>
<h1 id="identifying-molecules-as-biosignatures-with-assembly-theory-and-mass-spectrometry"><a aria-hidden="true" class="anchor-heading" href="#identifying-molecules-as-biosignatures-with-assembly-theory-and-mass-spectrometry"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Identifying molecules as biosignatures with assembly theory and mass spectrometry</h1>
<p><a href="https://www.nature.com/articles/s41467-021-23258-x">https://www.nature.com/articles/s41467-021-23258-x</a></p>
<blockquote>
<p>The search for alien life is hard because we do not know what signatures are unique to life. We show why complex molecules found in high abundance are universal biosignatures and demonstrate the first intrinsic experimentally tractable measure of molecular complexity, called the molecular assembly index (MA). To do this we calculate the complexity of several million molecules and validate that their complexity can be experimentally determined by mass spectrometry. This approach allows us to identify molecular biosignatures from a set of diverse samples from around the world, outer space, and the laboratory, demonstrating it is possible to build a life detection experiment based on MA that could be deployed to extraterrestrial locations, and used as a complexity scale to quantify constraints needed to direct prebiotically plausible processes in the laboratory. Such an approach is vital for finding life elsewhere in the universe or creating de-novo life in the lab.</p>
</blockquote>
<h1 id="supervised-topic-modeling-for-predicting-molecular-substructure-from-mass-spectrometry-version-1-peer-review-awaiting-peer-review"><a aria-hidden="true" class="anchor-heading" href="#supervised-topic-modeling-for-predicting-molecular-substructure-from-mass-spectrometry-version-1-peer-review-awaiting-peer-review"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Supervised topic modeling for predicting molecular substructure from mass spectrometry [version 1; peer review: awaiting peer review]</h1>
<p><a href="https://f1000research.com/articles/10-403">https://f1000research.com/articles/10-403</a></p>
<blockquote>
<p>Small-molecule metabolites are principal actors in myriad phenomena
across biochemistry and serve as an important source of biomarkers
and drug candidates. Given a sample of unknown composition,
identifying the metabolites present is difficult given the large number
of small molecules both known and yet to be discovered. Even for
biofluids such as human blood, building reliable ways of identifying
biomarkers is challenging. A workhorse method for characterizing
individual molecules in such untargeted metabolomics studies is
tandem mass spectrometry (MS/MS). MS/MS spectra provide rich
information about chemical composition. However, structural
characterization from spectra corresponding to unknown molecules
remains a bottleneck in metabolomics. Current methods often rely on
matching to pre-existing databases in one form or another.  Here we
develop a preprocessing scheme and supervised topic modeling
approach to identify modular groups of spectrum fragments and
neutral losses corresponding to chemical substructures using labeled
latent Dirichlet allocation (LLDA) to map spectrum features to known
chemical structures. These structures appear in new unknown spectra
and can be predicted. We find that LLDA is an interpretable and
reliable method for structure prediction from MS/MS spectra.
Specifically, the LLDA approach has the following advantages: (a)
molecular topics are interpretable; (b) A practitioner can select any set
of chemical structure labels relevant to their problem; (c ) LLDA
performs well and can exceed the performance of other methods in
predicting substructures in novel contexts.</p>
</blockquote>
<p>Tuesday 11 May 2021</p>
<h1 id="plant-scientists-research-attention-is-skewed-towards-colourful-conspicuous-and-broadly-distributed-flowers"><a aria-hidden="true" class="anchor-heading" href="#plant-scientists-research-attention-is-skewed-towards-colourful-conspicuous-and-broadly-distributed-flowers"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Plant scientists’ research attention is skewed towards colourful, conspicuous and broadly distributed flowers</h1>
<p><a href="https://www.nature.com/articles/s41477-021-00912-2">https://www.nature.com/articles/s41477-021-00912-2</a></p>
<p>Tuesday 20 April 2021</p>
<h1 id="repository-scale-classification-and-decomposition-of-tandem-mass-spectral-data"><a aria-hidden="true" class="anchor-heading" href="#repository-scale-classification-and-decomposition-of-tandem-mass-spectral-data"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Repository scale classification and decomposition of tandem mass spectral data</h1>
<p>Scientific Reports volume 11, Article number: 8314 (2021) Cite this article</p>
<p><a href="https://www.nature.com/articles/s41598-021-87796-6">https://www.nature.com/articles/s41598-021-87796-6</a></p>
<p>Monday 29 March 2021</p>
<h1 id="capturing-scientific-knowledge-in-computable-form"><a aria-hidden="true" class="anchor-heading" href="#capturing-scientific-knowledge-in-computable-form"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Capturing scientific knowledge in computable form</h1>
<p><a href="https://biofactoid.org">https://biofactoid.org</a></p>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.03.10.382333v1">https://www.biorxiv.org/content/10.1101/2021.03.10.382333v1</a></p>
<h1 id="first-known-gene-transfer-from-plant-to-insect-identified"><a aria-hidden="true" class="anchor-heading" href="#first-known-gene-transfer-from-plant-to-insect-identified"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>First known gene transfer from plant to insect identified</h1>
<p><a href="https://www.nature.com/articles/d41586-021-00782-w">https://www.nature.com/articles/d41586-021-00782-w</a></p>
<p>Friday 05 March 2021</p>
<h1 id="connecting-molecular-sequences-to-their-voucher-specimens"><a aria-hidden="true" class="anchor-heading" href="#connecting-molecular-sequences-to-their-voucher-specimens"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Connecting molecular sequences to their voucher specimens</h1>
<p><a href="https://biohackrxiv.org/93qf4/">https://biohackrxiv.org/93qf4/</a></p>
<p><a title="Private" style="color: brown" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank">#wikidata (Private)</a></p>
<p>2021-02-10 10:13</p>
<h1 id="large-scale-tandem-mass-spectrum-clustering-using-fast-nearest-neighbor-searching"><a aria-hidden="true" class="anchor-heading" href="#large-scale-tandem-mass-spectrum-clustering-using-fast-nearest-neighbor-searching"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Large-scale tandem mass spectrum clustering using fast nearest neighbor searching</h1>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.02.05.429957v1">https://www.biorxiv.org/content/10.1101/2021.02.05.429957v1</a></p>
<p><a title="Private" style="color: brown" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank">#spectral_comparison (Private)</a></p>
<p>2021-02-09 11:45</p>
<h1 id="netlsd-hearing-the-shape-of-a-graph"><a aria-hidden="true" class="anchor-heading" href="#netlsd-hearing-the-shape-of-a-graph"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>NetLSD: Hearing the Shape of a Graph</h1>
<p>Graph comparison an analysis at large scale</p>
<p><a href="https://arxiv.org/pdf/1805.10712.pdf">https://arxiv.org/pdf/1805.10712.pdf</a></p>
<p><a title="Private" style="color: brown" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank">#graph (Private)</a></p>
<p>2021-02-08 19:42</p>
<h1 id="automated-assembly-of-a-reference-taxonomy-for-phylogenetic-data-synthesis"><a aria-hidden="true" class="anchor-heading" href="#automated-assembly-of-a-reference-taxonomy-for-phylogenetic-data-synthesis"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Automated assembly of a reference taxonomy for phylogenetic data synthesis</h1>
<p>The original OTOL paper
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515096/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515096/</a></p>
<p><a title="Private" style="color: brown" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank">#taxo (Private)</a> </p>
<p>2021-02-03 16:51</p>
<h1 id="data-integration-enables-global-biodiversity-synthesis"><a aria-hidden="true" class="anchor-heading" href="#data-integration-enables-global-biodiversity-synthesis"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Data integration enables global biodiversity synthesis</h1>
<p><a href="https://www.pnas.org/content/118/6/e2018093118">https://www.pnas.org/content/118/6/e2018093118</a></p>
<p><a title="Private" style="color: brown" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank">#biodiversity (Private)</a> <a title="Private" style="color: brown" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank">#gbif (Private)</a></p>
<p>2021-01-31 20:19</p>
<h1 id="interoperable-chemical-structure-search-service"><a aria-hidden="true" class="anchor-heading" href="#interoperable-chemical-structure-search-service"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Interoperable chemical structure search service</h1>
<p><a href="https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0367-2">https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0367-2</a></p>
<p>2021-01-29 19:22</p>
<h1 id="mapping-the-space-of-chemical-reactions-using-attention-based-neural-networks"><a aria-hidden="true" class="anchor-heading" href="#mapping-the-space-of-chemical-reactions-using-attention-based-neural-networks"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Mapping the space of chemical reactions using attention-based neural networks</h1>
<p>Reymond's paper on reactions based tmaps. Try this on BNICE set of react or on MetaNetX</p>
<p><a href="https://www.nature.com/articles/s42256-020-00284-w.epdf?sharing_token=CGHu6tcqBiFmC1nRfnOrxNRgN0jAjWel9jnR3ZoTv0NiJSBmQfyOiLfbwen1TgEszNU5Xao_0Gs7D0u8tLUOJxaLAUc469WZKWG5K9wsqIe98dZOREVhQ33gqcr33AgNUjnp4cVXojp4aKe2xCbWtNaxOZtpOjBOsdX3O8yxVkI%3D">https://www.nature.com/articles/s42256-020-00284-w.epdf?sharing_token=CGHu6tcqBiFmC1nRfnOrxNRgN0jAjWel9jnR3ZoTv0NiJSBmQfyOiLfbwen1TgEszNU5Xao_0Gs7D0u8tLUOJxaLAUc469WZKWG5K9wsqIe98dZOREVhQ33gqcr33AgNUjnp4cVXojp4aKe2xCbWtNaxOZtpOjBOsdX3O8yxVkI%3D</a></p>
<p>2021-01-24 17:04</p>
<p># Estimate Metabolite Taxonomy and Structure with a FragmentCentered Database and Fragment Network</p>
<p><a href="https://arxiv.org/pdf/2101.03784.pdf">https://arxiv.org/pdf/2101.03784.pdf</a></p>
<p>2021-01-17 09:21</p>
<h1 id="network-based-strategies-in-metabolomics-data-analysis-and-interpretation-from-molecular-networking-to-biological-interpretation"><a aria-hidden="true" class="anchor-heading" href="#network-based-strategies-in-metabolomics-data-analysis-and-interpretation-from-molecular-networking-to-biological-interpretation"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Network-based strategies in metabolomics data analysis and interpretation: from molecular networking to biological interpretation</h1>
<p><a href="https://www.tandfonline.com/doi/full/10.1080/14789450.2020.1766975">https://www.tandfonline.com/doi/full/10.1080/14789450.2020.1766975</a></p>
<p><a title="Private" style="color: brown" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank">networks (Private)</a></p>
<p>2021-01-13 22:12</p>
<h1 id="towards-a-biodiversity-knowledge-graph"><a aria-hidden="true" class="anchor-heading" href="#towards-a-biodiversity-knowledge-graph"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Towards a biodiversity knowledge graph</h1>
<p><a href="https://riojournal.com/articles.php?journal_name=rio&#x26;id=8767">https://riojournal.com/articles.php?journal_name=rio&#x26;id=8767</a>
Check RIO in general
Dig into <a href="http://beta.briefideas.org/">http://beta.briefideas.org/</a>
//TODO Make a nanopub and related  note</p>
<p>2021-01-12 16:41</p>
<p># Metabolite networks filtering and explo</p>
<p><a href="https://www.biorxiv.org/content/biorxiv/early/2021/01/06/2021.01.06.425569.full.pdf">https://www.biorxiv.org/content/biorxiv/early/2021/01/06/2021.01.06.425569.full.pdf</a></p>
<p>2021-01-09 14:18</p>
<h1 id="florian-huber-blog-post-msmsmatch-tuto"><a aria-hidden="true" class="anchor-heading" href="#florian-huber-blog-post-msmsmatch-tuto"><svg aria-hidden="true" viewBox="0 0 16 16"><use xlink:href="#svg-link"></use></svg></a>Florian Huber blog post msmsmatch tuto</h1>
<p><a href="https://blog.esciencecenter.nl/build-your-own-mass-spectrometry-analysis-pipeline-in-python-using-matchms-part-i-d96c718c68ee">https://blog.esciencecenter.nl/build-your-own-mass-spectrometry-analysis-pipeline-in-python-using-matchms-part-i-d96c718c68ee</a></p>
<p>2021-01-09 13:15</p>
<p># Chemical graph generators
<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008504">https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008504</a></p>
<p>This resource represents an important advancement in the design and deployment of a comprehensive and collaborative natural products knowledge base.</p></div></div><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-0 ant-col-md-6"><div><div class=""><div class="ant-anchor-wrapper dendron-toc" style="max-height:calc(100vh - 64px);z-index:1"><div class="ant-anchor"><div class="ant-anchor-ink"><span class="ant-anchor-ink-ball"></span></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#readlist-of-publications" title="Readlist of publications">Readlist of publications</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#a-knowledge-graph-to-interpret-clinical-proteomics-data" title="A knowledge graph to interpret clinical proteomics data">A knowledge graph to interpret clinical proteomics data</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#infrastructure-and-population-of-the-openbiodiv-biodiversity-knowledge-graph" title="Infrastructure and Population of the OpenBiodiv Biodiversity Knowledge Graph">Infrastructure and Population of the OpenBiodiv Biodiversity Knowledge Graph</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#reshaping-the-academic-self-connecting-education--open-science" title="Reshaping the Academic Self: Connecting Education &amp; Open Science">Reshaping the Academic Self: Connecting Education &amp; Open Science</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#functional-biology-in-its-natural-context-a-search-for-emergent-simplicity" title="Functional biology in its natural context: A search for emergent simplicity">Functional biology in its natural context: A search for emergent simplicity</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#a-deep-generative-model-enables-automated-structure-elucidation-of-novel-psychoactive-substances" title="A deep generative model enables automated structure elucidation of novel psychoactive substances">A deep generative model enables automated structure elucidation of novel psychoactive substances</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#multiomic-big-data-analysis-challenges-increasing-confidence-in-the-interpretation-of-artificial-intelligence-assessments" title="Multiomic Big Data Analysis Challenges: Increasing Confidence in the Interpretation of Artificial Intelligence Assessments">Multiomic Big Data Analysis Challenges: Increasing Confidence in the Interpretation of Artificial Intelligence Assessments</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#identifying-molecules-as-biosignatures-with-assembly-theory-and-mass-spectrometry" title="Identifying molecules as biosignatures with assembly theory and mass spectrometry">Identifying molecules as biosignatures with assembly theory and mass spectrometry</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#supervised-topic-modeling-for-predicting-molecular-substructure-from-mass-spectrometry-version-1-peer-review-awaiting-peer-review" title="Supervised topic modeling for predicting molecular substructure from mass spectrometry version 1; peer review: awaiting peer review">Supervised topic modeling for predicting molecular substructure from mass spectrometry version 1; peer review: awaiting peer review</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#plant-scientists-research-attention-is-skewed-towards-colourful-conspicuous-and-broadly-distributed-flowers" title="Plant scientists’ research attention is skewed towards colourful, conspicuous and broadly distributed flowers">Plant scientists’ research attention is skewed towards colourful, conspicuous and broadly distributed flowers</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#repository-scale-classification-and-decomposition-of-tandem-mass-spectral-data" title="Repository scale classification and decomposition of tandem mass spectral data">Repository scale classification and decomposition of tandem mass spectral data</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#capturing-scientific-knowledge-in-computable-form" title="Capturing scientific knowledge in computable form">Capturing scientific knowledge in computable form</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#first-known-gene-transfer-from-plant-to-insect-identified" title="First known gene transfer from plant to insect identified">First known gene transfer from plant to insect identified</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#connecting-molecular-sequences-to-their-voucher-specimens" title="Connecting molecular sequences to their voucher specimens">Connecting molecular sequences to their voucher specimens</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#large-scale-tandem-mass-spectrum-clustering-using-fast-nearest-neighbor-searching" title="Large Scale tandem mass spectrum clustering using fast nearest neighbor searching">Large Scale tandem mass spectrum clustering using fast nearest neighbor searching</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#netlsd-hearing-the-shape-of-a-graph" title="NetLSD: Hearing the Shape of a Graph">NetLSD: Hearing the Shape of a Graph</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#automated-assembly-of-a-reference-taxonomy-for-phylogenetic-data-synthesis" title="Automated assembly of a reference taxonomy for phylogenetic data synthesis">Automated assembly of a reference taxonomy for phylogenetic data synthesis</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#data-integration-enables-global-biodiversity-synthesis" title="Data integration enables global biodiversity synthesis">Data integration enables global biodiversity synthesis</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#interoperable-chemical-structure-search-service" title="Interoperable chemical structure search service">Interoperable chemical structure search service</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#mapping-the-space-of-chemical-reactions-using-attention-based-neural-networks" title="Mapping the space of chemical reactions using attention Based neural networks">Mapping the space of chemical reactions using attention Based neural networks</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#network-based-strategies-in-metabolomics-data-analysis-and-interpretation-from-molecular-networking-to-biological-interpretation" title="Network Based strategies in metabolomics data analysis and interpretation: from molecular networking to biological interpretation">Network Based strategies in metabolomics data analysis and interpretation: from molecular networking to biological interpretation</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#towards-a-biodiversity-knowledge-graph" title="Towards a biodiversity knowledge graph">Towards a biodiversity knowledge graph</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#florian-huber-blog-post-msmsmatch-tuto" title="Florian Huber blog post msmsmatch tuto">Florian Huber blog post msmsmatch tuto</a></div></div></div></div></div></div></div></div></div></main><div class="ant-divider ant-divider-horizontal" role="separator"></div><footer class="ant-layout-footer" style="padding:0 24px 24px"></footer></section></section></section></section></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"note":{"id":"3PbjxNe7yIn2I9yKJfqXD","title":"Papers","desc":"","updated":1643739381213,"created":1610194509817,"custom":{},"fname":"readlist.papers","type":"note","vault":{"fsPath":"vault"},"contentHash":"6c6e4488b090c98179e6bb60190d7ef7","links":[{"type":"wiki","from":{"fname":"readlist.papers","id":"3PbjxNe7yIn2I9yKJfqXD","vaultName":"vault"},"value":"tag.wikidata","alias":"#wikidata","position":{"start":{"line":125,"column":1,"offset":4929},"end":{"line":125,"column":27,"offset":4955},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"tag.wikidata"}},{"type":"wiki","from":{"fname":"readlist.papers","id":"3PbjxNe7yIn2I9yKJfqXD","vaultName":"vault"},"value":"tag.spectral_comparison","alias":"#spectral_comparison","position":{"start":{"line":135,"column":1,"offset":5123},"end":{"line":135,"column":49,"offset":5171},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"tag.spectral_comparison"}},{"type":"wiki","from":{"fname":"readlist.papers","id":"3PbjxNe7yIn2I9yKJfqXD","vaultName":"vault"},"value":"tag.graph","alias":"#graph","position":{"start":{"line":145,"column":1,"offset":5314},"end":{"line":145,"column":21,"offset":5334},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"tag.graph"}},{"type":"wiki","from":{"fname":"readlist.papers","id":"3PbjxNe7yIn2I9yKJfqXD","vaultName":"vault"},"value":"tag.taxo","alias":"#taxo","position":{"start":{"line":155,"column":1,"offset":5512},"end":{"line":155,"column":19,"offset":5530},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"tag.taxo"}},{"type":"wiki","from":{"fname":"readlist.papers","id":"3PbjxNe7yIn2I9yKJfqXD","vaultName":"vault"},"value":"tag.biodiversity","alias":"#biodiversity","position":{"start":{"line":164,"column":1,"offset":5658},"end":{"line":164,"column":35,"offset":5692},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"tag.biodiversity"}},{"type":"wiki","from":{"fname":"readlist.papers","id":"3PbjxNe7yIn2I9yKJfqXD","vaultName":"vault"},"value":"tag.gbif","alias":"#gbif","position":{"start":{"line":164,"column":36,"offset":5693},"end":{"line":164,"column":54,"offset":5711},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"tag.gbif"}},{"type":"wiki","from":{"fname":"readlist.papers","id":"3PbjxNe7yIn2I9yKJfqXD","vaultName":"vault"},"value":"networks","alias":"networks","position":{"start":{"line":203,"column":1,"offset":6673},"end":{"line":203,"column":13,"offset":6685},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"networks"}}],"anchors":{"readlist-of-publications":{"type":"header","text":"Readlist of publications","value":"readlist-of-publications","line":9,"column":0,"depth":1},"a-knowledge-graph-to-interpret-clinical-proteomics-data":{"type":"header","text":"A knowledge graph to interpret clinical proteomics data","value":"a-knowledge-graph-to-interpret-clinical-proteomics-data","line":16,"column":0,"depth":1},"infrastructure-and-population-of-the-openbiodiv-biodiversity-knowledge-graph":{"type":"header","text":"Infrastructure and Population of the OpenBiodiv Biodiversity Knowledge Graph","value":"infrastructure-and-population-of-the-openbiodiv-biodiversity-knowledge-graph","line":21,"column":0,"depth":1},"reshaping-the-academic-self-connecting-education--open-science":{"type":"header","text":"Reshaping the Academic Self: Connecting Education \u0026 Open Science","value":"reshaping-the-academic-self-connecting-education--open-science","line":26,"column":0,"depth":1},"functional-biology-in-its-natural-context-a-search-for-emergent-simplicity":{"type":"header","text":"Functional biology in its natural context: A search for emergent simplicity","value":"functional-biology-in-its-natural-context-a-search-for-emergent-simplicity","line":36,"column":0,"depth":1},"a-deep-generative-model-enables-automated-structure-elucidation-of-novel-psychoactive-substances":{"type":"header","text":"A deep generative model enables automated structure elucidation of novel psychoactive substances","value":"a-deep-generative-model-enables-automated-structure-elucidation-of-novel-psychoactive-substances","line":42,"column":0,"depth":1},"multiomic-big-data-analysis-challenges-increasing-confidence-in-the-interpretation-of-artificial-intelligence-assessments":{"type":"header","text":"Multiomic Big Data Analysis Challenges: Increasing Confidence in the Interpretation of Artificial Intelligence Assessments","value":"multiomic-big-data-analysis-challenges-increasing-confidence-in-the-interpretation-of-artificial-intelligence-assessments","line":49,"column":0,"depth":1},"identifying-molecules-as-biosignatures-with-assembly-theory-and-mass-spectrometry":{"type":"header","text":"Identifying molecules as biosignatures with assembly theory and mass spectrometry","value":"identifying-molecules-as-biosignatures-with-assembly-theory-and-mass-spectrometry","line":56,"column":0,"depth":1},"supervised-topic-modeling-for-predicting-molecular-substructure-from-mass-spectrometry-version-1-peer-review-awaiting-peer-review":{"type":"header","text":"Supervised topic modeling for predicting molecular substructure from mass spectrometry version 1; peer review: awaiting peer review","value":"supervised-topic-modeling-for-predicting-molecular-substructure-from-mass-spectrometry-version-1-peer-review-awaiting-peer-review","line":62,"column":0,"depth":1},"plant-scientists-research-attention-is-skewed-towards-colourful-conspicuous-and-broadly-distributed-flowers":{"type":"header","text":"Plant scientists’ research attention is skewed towards colourful, conspicuous and broadly distributed flowers","value":"plant-scientists-research-attention-is-skewed-towards-colourful-conspicuous-and-broadly-distributed-flowers","line":95,"column":0,"depth":1},"repository-scale-classification-and-decomposition-of-tandem-mass-spectral-data":{"type":"header","text":"Repository scale classification and decomposition of tandem mass spectral data","value":"repository-scale-classification-and-decomposition-of-tandem-mass-spectral-data","line":102,"column":0,"depth":1},"capturing-scientific-knowledge-in-computable-form":{"type":"header","text":"Capturing scientific knowledge in computable form","value":"capturing-scientific-knowledge-in-computable-form","line":110,"column":0,"depth":1},"first-known-gene-transfer-from-plant-to-insect-identified":{"type":"header","text":"First known gene transfer from plant to insect identified","value":"first-known-gene-transfer-from-plant-to-insect-identified","line":118,"column":0,"depth":1},"connecting-molecular-sequences-to-their-voucher-specimens":{"type":"header","text":"Connecting molecular sequences to their voucher specimens","value":"connecting-molecular-sequences-to-their-voucher-specimens","line":127,"column":0,"depth":1},"large-scale-tandem-mass-spectrum-clustering-using-fast-nearest-neighbor-searching":{"type":"header","text":"Large-scale tandem mass spectrum clustering using fast nearest neighbor searching","value":"large-scale-tandem-mass-spectrum-clustering-using-fast-nearest-neighbor-searching","line":136,"column":0,"depth":1},"netlsd-hearing-the-shape-of-a-graph":{"type":"header","text":"NetLSD: Hearing the Shape of a Graph","value":"netlsd-hearing-the-shape-of-a-graph","line":145,"column":0,"depth":1},"automated-assembly-of-a-reference-taxonomy-for-phylogenetic-data-synthesis":{"type":"header","text":"Automated assembly of a reference taxonomy for phylogenetic data synthesis","value":"automated-assembly-of-a-reference-taxonomy-for-phylogenetic-data-synthesis","line":156,"column":0,"depth":1},"data-integration-enables-global-biodiversity-synthesis":{"type":"header","text":"Data integration enables global biodiversity synthesis","value":"data-integration-enables-global-biodiversity-synthesis","line":166,"column":0,"depth":1},"interoperable-chemical-structure-search-service":{"type":"header","text":"Interoperable chemical structure search service","value":"interoperable-chemical-structure-search-service","line":175,"column":0,"depth":1},"mapping-the-space-of-chemical-reactions-using-attention-based-neural-networks":{"type":"header","text":"Mapping the space of chemical reactions using attention-based neural networks","value":"mapping-the-space-of-chemical-reactions-using-attention-based-neural-networks","line":184,"column":0,"depth":1},"network-based-strategies-in-metabolomics-data-analysis-and-interpretation-from-molecular-networking-to-biological-interpretation":{"type":"header","text":"Network-based strategies in metabolomics data analysis and interpretation: from molecular networking to biological interpretation","value":"network-based-strategies-in-metabolomics-data-analysis-and-interpretation-from-molecular-networking-to-biological-interpretation","line":205,"column":0,"depth":1},"towards-a-biodiversity-knowledge-graph":{"type":"header","text":"Towards a biodiversity knowledge graph","value":"towards-a-biodiversity-knowledge-graph","line":216,"column":0,"depth":1},"florian-huber-blog-post-msmsmatch-tuto":{"type":"header","text":"Florian Huber blog post msmsmatch tuto","value":"florian-huber-blog-post-msmsmatch-tuto","line":233,"column":0,"depth":1}},"children":[],"parent":"iRR8MxJPGIvfKl5pcOirA","data":{}},"body":"\u003ch1 id=\"papers\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#papers\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003ePapers\u003c/h1\u003e\n\u003ch1 id=\"readlist-of-publications\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#readlist-of-publications\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eReadlist of publications\u003c/h1\u003e\n\u003cp\u003eThis is a dump.\nEntries here should be sent to their respective dendron notes when treated.\u003c/p\u003e\n\u003ch1 id=\"a-knowledge-graph-to-interpret-clinical-proteomics-data\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#a-knowledge-graph-to-interpret-clinical-proteomics-data\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eA knowledge graph to interpret clinical proteomics data\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.nature.com/articles/s41587-021-01145-6\"\u003ehttps://www.nature.com/articles/s41587-021-01145-6\u003c/a\u003e\u003c/p\u003e\n\u003ch1 id=\"infrastructure-and-population-of-the-openbiodiv-biodiversity-knowledge-graph\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#infrastructure-and-population-of-the-openbiodiv-biodiversity-knowledge-graph\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eInfrastructure and Population of the OpenBiodiv Biodiversity Knowledge Graph\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://bdj.pensoft.net/article/67671/\"\u003ehttps://bdj.pensoft.net/article/67671/\u003c/a\u003e\u003c/p\u003e\n\u003ch1 id=\"reshaping-the-academic-self-connecting-education--open-science\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#reshaping-the-academic-self-connecting-education--open-science\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eReshaping the Academic Self: Connecting Education \u0026#x26; Open Science\u003c/h1\u003e\n\u003cp\u003eSunday 05 September 2021\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"https://zenodo.org/record/5345573#.YTSAwDZfiDW\"\u003ehttps://zenodo.org/record/5345573#.YTSAwDZfiDW\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eMonday 14 June 2021\u003c/p\u003e\n\u003ch1 id=\"functional-biology-in-its-natural-context-a-search-for-emergent-simplicity\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#functional-biology-in-its-natural-context-a-search-for-emergent-simplicity\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eFunctional biology in its natural context: A search for emergent simplicity\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://elifesciences.org/articles/67646\"\u003ehttps://elifesciences.org/articles/67646\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eMonday 31 May 2021\u003c/p\u003e\n\u003ch1 id=\"a-deep-generative-model-enables-automated-structure-elucidation-of-novel-psychoactive-substances\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#a-deep-generative-model-enables-automated-structure-elucidation-of-novel-psychoactive-substances\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eA deep generative model enables automated structure elucidation of novel psychoactive substances\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://chemrxiv.org/articles/preprint/A_Deep_Generative_Model_Enables_Automated_Structure_Elucidation_of_Novel_Psychoactive_Substances/14644854/1\"\u003ehttps://chemrxiv.org/articles/preprint/A_Deep_Generative_Model_Enables_Automated_Structure_Elucidation_of_Novel_Psychoactive_Substances/14644854/1\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eThursday 27 May 2021\u003c/p\u003e\n\u003ch1 id=\"multiomic-big-data-analysis-challenges-increasing-confidence-in-the-interpretation-of-artificial-intelligence-assessments\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#multiomic-big-data-analysis-challenges-increasing-confidence-in-the-interpretation-of-artificial-intelligence-assessments\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eMultiomic Big Data Analysis Challenges: Increasing Confidence in the Interpretation of Artificial Intelligence Assessments\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c04850\"\u003ehttps://pubs.acs.org/doi/abs/10.1021/acs.analchem.0c04850\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eTuesday 25 May 2021\u003c/p\u003e\n\u003ch1 id=\"identifying-molecules-as-biosignatures-with-assembly-theory-and-mass-spectrometry\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#identifying-molecules-as-biosignatures-with-assembly-theory-and-mass-spectrometry\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eIdentifying molecules as biosignatures with assembly theory and mass spectrometry\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.nature.com/articles/s41467-021-23258-x\"\u003ehttps://www.nature.com/articles/s41467-021-23258-x\u003c/a\u003e\u003c/p\u003e\n\u003cblockquote\u003e\n\u003cp\u003eThe search for alien life is hard because we do not know what signatures are unique to life. We show why complex molecules found in high abundance are universal biosignatures and demonstrate the first intrinsic experimentally tractable measure of molecular complexity, called the molecular assembly index (MA). To do this we calculate the complexity of several million molecules and validate that their complexity can be experimentally determined by mass spectrometry. This approach allows us to identify molecular biosignatures from a set of diverse samples from around the world, outer space, and the laboratory, demonstrating it is possible to build a life detection experiment based on MA that could be deployed to extraterrestrial locations, and used as a complexity scale to quantify constraints needed to direct prebiotically plausible processes in the laboratory. Such an approach is vital for finding life elsewhere in the universe or creating de-novo life in the lab.\u003c/p\u003e\n\u003c/blockquote\u003e\n\u003ch1 id=\"supervised-topic-modeling-for-predicting-molecular-substructure-from-mass-spectrometry-version-1-peer-review-awaiting-peer-review\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#supervised-topic-modeling-for-predicting-molecular-substructure-from-mass-spectrometry-version-1-peer-review-awaiting-peer-review\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eSupervised topic modeling for predicting molecular substructure from mass spectrometry [version 1; peer review: awaiting peer review]\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://f1000research.com/articles/10-403\"\u003ehttps://f1000research.com/articles/10-403\u003c/a\u003e\u003c/p\u003e\n\u003cblockquote\u003e\n\u003cp\u003eSmall-molecule metabolites are principal actors in myriad phenomena\nacross biochemistry and serve as an important source of biomarkers\nand drug candidates. Given a sample of unknown composition,\nidentifying the metabolites present is difficult given the large number\nof small molecules both known and yet to be discovered. Even for\nbiofluids such as human blood, building reliable ways of identifying\nbiomarkers is challenging. A workhorse method for characterizing\nindividual molecules in such untargeted metabolomics studies is\ntandem mass spectrometry (MS/MS). MS/MS spectra provide rich\ninformation about chemical composition. However, structural\ncharacterization from spectra corresponding to unknown molecules\nremains a bottleneck in metabolomics. Current methods often rely on\nmatching to pre-existing databases in one form or another.  Here we\ndevelop a preprocessing scheme and supervised topic modeling\napproach to identify modular groups of spectrum fragments and\nneutral losses corresponding to chemical substructures using labeled\nlatent Dirichlet allocation (LLDA) to map spectrum features to known\nchemical structures. These structures appear in new unknown spectra\nand can be predicted. We find that LLDA is an interpretable and\nreliable method for structure prediction from MS/MS spectra.\nSpecifically, the LLDA approach has the following advantages: (a)\nmolecular topics are interpretable; (b) A practitioner can select any set\nof chemical structure labels relevant to their problem; (c ) LLDA\nperforms well and can exceed the performance of other methods in\npredicting substructures in novel contexts.\u003c/p\u003e\n\u003c/blockquote\u003e\n\u003cp\u003eTuesday 11 May 2021\u003c/p\u003e\n\u003ch1 id=\"plant-scientists-research-attention-is-skewed-towards-colourful-conspicuous-and-broadly-distributed-flowers\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#plant-scientists-research-attention-is-skewed-towards-colourful-conspicuous-and-broadly-distributed-flowers\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003ePlant scientists’ research attention is skewed towards colourful, conspicuous and broadly distributed flowers\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.nature.com/articles/s41477-021-00912-2\"\u003ehttps://www.nature.com/articles/s41477-021-00912-2\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eTuesday 20 April 2021\u003c/p\u003e\n\u003ch1 id=\"repository-scale-classification-and-decomposition-of-tandem-mass-spectral-data\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#repository-scale-classification-and-decomposition-of-tandem-mass-spectral-data\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eRepository scale classification and decomposition of tandem mass spectral data\u003c/h1\u003e\n\u003cp\u003eScientific Reports volume 11, Article number: 8314 (2021) Cite this article\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"https://www.nature.com/articles/s41598-021-87796-6\"\u003ehttps://www.nature.com/articles/s41598-021-87796-6\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eMonday 29 March 2021\u003c/p\u003e\n\u003ch1 id=\"capturing-scientific-knowledge-in-computable-form\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#capturing-scientific-knowledge-in-computable-form\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eCapturing scientific knowledge in computable form\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://biofactoid.org\"\u003ehttps://biofactoid.org\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"https://www.biorxiv.org/content/10.1101/2021.03.10.382333v1\"\u003ehttps://www.biorxiv.org/content/10.1101/2021.03.10.382333v1\u003c/a\u003e\u003c/p\u003e\n\u003ch1 id=\"first-known-gene-transfer-from-plant-to-insect-identified\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#first-known-gene-transfer-from-plant-to-insect-identified\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eFirst known gene transfer from plant to insect identified\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.nature.com/articles/d41586-021-00782-w\"\u003ehttps://www.nature.com/articles/d41586-021-00782-w\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eFriday 05 March 2021\u003c/p\u003e\n\u003ch1 id=\"connecting-molecular-sequences-to-their-voucher-specimens\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#connecting-molecular-sequences-to-their-voucher-specimens\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eConnecting molecular sequences to their voucher specimens\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://biohackrxiv.org/93qf4/\"\u003ehttps://biohackrxiv.org/93qf4/\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca title=\"Private\" style=\"color: brown\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\"\u003e#wikidata (Private)\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-02-10 10:13\u003c/p\u003e\n\u003ch1 id=\"large-scale-tandem-mass-spectrum-clustering-using-fast-nearest-neighbor-searching\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#large-scale-tandem-mass-spectrum-clustering-using-fast-nearest-neighbor-searching\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eLarge-scale tandem mass spectrum clustering using fast nearest neighbor searching\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.biorxiv.org/content/10.1101/2021.02.05.429957v1\"\u003ehttps://www.biorxiv.org/content/10.1101/2021.02.05.429957v1\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca title=\"Private\" style=\"color: brown\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\"\u003e#spectral_comparison (Private)\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-02-09 11:45\u003c/p\u003e\n\u003ch1 id=\"netlsd-hearing-the-shape-of-a-graph\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#netlsd-hearing-the-shape-of-a-graph\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eNetLSD: Hearing the Shape of a Graph\u003c/h1\u003e\n\u003cp\u003eGraph comparison an analysis at large scale\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"https://arxiv.org/pdf/1805.10712.pdf\"\u003ehttps://arxiv.org/pdf/1805.10712.pdf\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca title=\"Private\" style=\"color: brown\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\"\u003e#graph (Private)\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-02-08 19:42\u003c/p\u003e\n\u003ch1 id=\"automated-assembly-of-a-reference-taxonomy-for-phylogenetic-data-synthesis\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#automated-assembly-of-a-reference-taxonomy-for-phylogenetic-data-synthesis\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eAutomated assembly of a reference taxonomy for phylogenetic data synthesis\u003c/h1\u003e\n\u003cp\u003eThe original OTOL paper\n\u003ca href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515096/\"\u003ehttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5515096/\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca title=\"Private\" style=\"color: brown\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\"\u003e#taxo (Private)\u003c/a\u003e \u003c/p\u003e\n\u003cp\u003e2021-02-03 16:51\u003c/p\u003e\n\u003ch1 id=\"data-integration-enables-global-biodiversity-synthesis\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#data-integration-enables-global-biodiversity-synthesis\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eData integration enables global biodiversity synthesis\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.pnas.org/content/118/6/e2018093118\"\u003ehttps://www.pnas.org/content/118/6/e2018093118\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca title=\"Private\" style=\"color: brown\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\"\u003e#biodiversity (Private)\u003c/a\u003e \u003ca title=\"Private\" style=\"color: brown\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\"\u003e#gbif (Private)\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-01-31 20:19\u003c/p\u003e\n\u003ch1 id=\"interoperable-chemical-structure-search-service\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#interoperable-chemical-structure-search-service\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eInteroperable chemical structure search service\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0367-2\"\u003ehttps://jcheminf.biomedcentral.com/articles/10.1186/s13321-019-0367-2\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-01-29 19:22\u003c/p\u003e\n\u003ch1 id=\"mapping-the-space-of-chemical-reactions-using-attention-based-neural-networks\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#mapping-the-space-of-chemical-reactions-using-attention-based-neural-networks\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eMapping the space of chemical reactions using attention-based neural networks\u003c/h1\u003e\n\u003cp\u003eReymond's paper on reactions based tmaps. Try this on BNICE set of react or on MetaNetX\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"https://www.nature.com/articles/s42256-020-00284-w.epdf?sharing_token=CGHu6tcqBiFmC1nRfnOrxNRgN0jAjWel9jnR3ZoTv0NiJSBmQfyOiLfbwen1TgEszNU5Xao_0Gs7D0u8tLUOJxaLAUc469WZKWG5K9wsqIe98dZOREVhQ33gqcr33AgNUjnp4cVXojp4aKe2xCbWtNaxOZtpOjBOsdX3O8yxVkI%3D\"\u003ehttps://www.nature.com/articles/s42256-020-00284-w.epdf?sharing_token=CGHu6tcqBiFmC1nRfnOrxNRgN0jAjWel9jnR3ZoTv0NiJSBmQfyOiLfbwen1TgEszNU5Xao_0Gs7D0u8tLUOJxaLAUc469WZKWG5K9wsqIe98dZOREVhQ33gqcr33AgNUjnp4cVXojp4aKe2xCbWtNaxOZtpOjBOsdX3O8yxVkI%3D\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-01-24 17:04\u003c/p\u003e\n\u003cp\u003e# Estimate Metabolite Taxonomy and Structure with a FragmentCentered Database and Fragment Network\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"https://arxiv.org/pdf/2101.03784.pdf\"\u003ehttps://arxiv.org/pdf/2101.03784.pdf\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-01-17 09:21\u003c/p\u003e\n\u003ch1 id=\"network-based-strategies-in-metabolomics-data-analysis-and-interpretation-from-molecular-networking-to-biological-interpretation\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#network-based-strategies-in-metabolomics-data-analysis-and-interpretation-from-molecular-networking-to-biological-interpretation\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eNetwork-based strategies in metabolomics data analysis and interpretation: from molecular networking to biological interpretation\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://www.tandfonline.com/doi/full/10.1080/14789450.2020.1766975\"\u003ehttps://www.tandfonline.com/doi/full/10.1080/14789450.2020.1766975\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca title=\"Private\" style=\"color: brown\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\"\u003enetworks (Private)\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-01-13 22:12\u003c/p\u003e\n\u003ch1 id=\"towards-a-biodiversity-knowledge-graph\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#towards-a-biodiversity-knowledge-graph\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eTowards a biodiversity knowledge graph\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://riojournal.com/articles.php?journal_name=rio\u0026#x26;id=8767\"\u003ehttps://riojournal.com/articles.php?journal_name=rio\u0026#x26;id=8767\u003c/a\u003e\nCheck RIO in general\nDig into \u003ca href=\"http://beta.briefideas.org/\"\u003ehttp://beta.briefideas.org/\u003c/a\u003e\n//TODO Make a nanopub and related  note\u003c/p\u003e\n\u003cp\u003e2021-01-12 16:41\u003c/p\u003e\n\u003cp\u003e# Metabolite networks filtering and explo\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"https://www.biorxiv.org/content/biorxiv/early/2021/01/06/2021.01.06.425569.full.pdf\"\u003ehttps://www.biorxiv.org/content/biorxiv/early/2021/01/06/2021.01.06.425569.full.pdf\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-01-09 14:18\u003c/p\u003e\n\u003ch1 id=\"florian-huber-blog-post-msmsmatch-tuto\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading\" href=\"#florian-huber-blog-post-msmsmatch-tuto\"\u003e\u003csvg aria-hidden=\"true\" viewBox=\"0 0 16 16\"\u003e\u003cuse xlink:href=\"#svg-link\"\u003e\u003c/use\u003e\u003c/svg\u003e\u003c/a\u003eFlorian Huber blog post msmsmatch tuto\u003c/h1\u003e\n\u003cp\u003e\u003ca href=\"https://blog.esciencecenter.nl/build-your-own-mass-spectrometry-analysis-pipeline-in-python-using-matchms-part-i-d96c718c68ee\"\u003ehttps://blog.esciencecenter.nl/build-your-own-mass-spectrometry-analysis-pipeline-in-python-using-matchms-part-i-d96c718c68ee\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e2021-01-09 13:15\u003c/p\u003e\n\u003cp\u003e# Chemical graph generators\n\u003ca href=\"https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008504\"\u003ehttps://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008504\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eThis resource represents an important advancement in the design and deployment of a comprehensive and collaborative natural products knowledge base.\u003c/p\u003e","noteIndex":{"id":"gvOH1py1NfW3ihejtp1nt","title":"Root","desc":"","updated":1629628449617,"created":1629628449617,"custom":{"nav_order":0,"permalink":"/"},"fname":"root","type":"note","vault":{"fsPath":"vault"},"contentHash":"35415410233b3af68001893c83fd0c46","links":[{"type":"wiki","from":{"fname":"root","id":"gvOH1py1NfW3ihejtp1nt","vaultName":"vault"},"value":"user.oolonek","alias":"@oolonek","position":{"start":{"line":5,"column":7,"offset":225},"end":{"line":5,"column":15,"offset":233},"indent":[]},"xvault":false,"to":{"fname":"user.oolonek"}}],"anchors":{"welcome-to-dendron":{"type":"header","text":"Welcome to Dendron","value":"welcome-to-dendron","line":7,"column":0,"depth":1}},"children":["kPtYPP8JzWBTzWz6JpnkS","3LmAI71UuDJHeWZiY9h96","TyTThFOL2RNC8FAIj5bZN","e4C8xkTTKKue3fz5ywgBH","rInb8zzM4kEHelz4CeWDa","hUOQrqfgNFdSu7GD8fP2c","J9JmfJgjDT8Oz6WilFnKw","Iu50F9KOgYp0CDpwKBlx6","xZShWyRUMIXQaN30pKmCM","f1fsD9inrKMVapeKNhEhT","VhvHTRFNvIEee7odQvekB","D2j8S1AufMK6GbZTZ9BJz","HMvEFZAnQOxD77TnbVwGV","NYpAWkhSLKKkHIbDIbshA","0MSZusMXhweDvHD3Iggww","q5kFqQrTnytdWSux3nwOk","1PGMuBcw56lKG0Xu5sxIX","f4P2xBJLTHpQpJYiJh7BV","7KJGsaBJWOOHaHqElRaJq","BunyVngKi3As0C0CO5xhy","iRR8MxJPGIvfKl5pcOirA","3qcjidzC0tkyobYKlrWYc","uviazqrM9NUAMm9dWYVa7","53WWGL5wGfRDGtBEJoqBd","EoQW6Rm5LYkkCxMgDVXHd","k7IyxWpd9nYqWlhZ96CAG","maJAMIUSNCqgSTXrQ7fu5","59ftsVxFXXM9bB43vfKwK","rrfMr3vOgkyJv21N8x99X","7WpN3kiuBsunRJ17P3mGr","u88hb42Nk8jvzQwtMfM3z"],"parent":null,"data":{},"body":"# Welcome to Dendron\n\nThis is the root of your dendron vault. If you decide to publish your entire vault, this will be your landing page. You are free to customize any part of this page except the frontmatter on top. \n\nKudos @oolonek, you did great job! \nWhen using https://oolonek.github.io/dendron-ws-private/, here is what I got:\n\n\u003cimg width=\"1676\" alt=\"Capture d’écran 2021-08-22 à 13 19 54\" src=\"https://user-images.githubusercontent.com/44283913/130353241-43ba98eb-d947-473c-be08-d924f0ef585b.png\"\u003e\n\n"},"collectionChildren":null,"customHeadContent":null,"config":{"version":4,"useFMTitle":true,"useNoteTitleForLink":true,"mermaid":true,"useKatex":true,"dev":{"enablePreviewV2":true},"site":{"copyAssets":true,"siteHierarchies":["root"],"siteRootDir":"docs","siteUrl":"https://oolonek.github.io","gh_edit_link":true,"gh_edit_link_text":"Click here to edit this page on Github !","gh_edit_repository":"https://github.com/oolonek/dendron-ws-public","gh_edit_branch":"main","gh_edit_view_mode":"edit","assetsPrefix":"/dendron-ws-public","usePrettyRefs":true,"siteLastModified":true,"title":"GrndStt's Dendron","description":"Personal knowledge space","duplicateNoteBehavior":{"action":"useVault","payload":["vault"]},"usePrettyLinks":true,"siteNotesDir":"notes","siteFaviconPath":"favicon.ico","gh_root":"docs/","writeStubs":true,"siteIndex":"root"},"commands":{"lookup":{"note":{"selectionMode":"extract","confirmVaultOnCreate":false,"leaveTrace":false,"bubbleUpCreateNew":true,"fuzzThreshold":0.2,"vaultSelectionModeOnCreate":"smart"}},"insertNote":{"initialValue":"templates"},"insertNoteLink":{"aliasMode":"none","enableMultiSelect":false},"insertNoteIndex":{"enableMarker":false},"randomNote":{}},"workspace":{"dendronVersion":"0.77.1","vaults":[{"fsPath":"vault"}],"journal":{"dailyDomain":"daily","name":"journal","dateFormat":"y.MM.dd","addBehavior":"childOfDomain"},"scratch":{"name":"scratch","dateFormat":"y.MM.dd.HHmmss","addBehavior":"asOwnDomain"},"graph":{"zoomSpeed":1},"enableAutoCreateOnDefinition":false,"enableXVaultWikiLink":false,"enableRemoteVaultInit":true,"workspaceVaultSyncMode":"noCommit","enableAutoFoldFrontmatter":true,"maxPreviewsCached":10,"maxNoteLength":204800,"task":{"name":"","dateFormat":"","addBehavior":"childOfCurrent","statusSymbols":{"":" ","wip":"w","done":"x","assigned":"a","moved":"m","blocked":"b","delegated":"l","dropped":"d","pending":"y"},"prioritySymbols":{"H":"high","M":"medium","L":"low"},"todoIntegration":false,"createTaskSelectionType":"selection2link"},"enableUserTags":true,"enableHashTags":true,"enableEditorDecorations":true},"preview":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enableMermaid":true,"enablePrettyRefs":true,"enableKatex":true,"automaticallyShowPreview":false}}},"__N_SSG":true},"page":"/notes/[id]","query":{"id":"3PbjxNe7yIn2I9yKJfqXD"},"buildId":"Fif_h8o_yalnGTkyFx1wu","assetPrefix":"/dendron-ws-public","isFallback":false,"gsp":true,"scriptLoader":[]}</script></body></html>